package.json

```diff
@@ -6,26 +6,30 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "@vitejs/plugin-react": "^4.3.3",
     "firebase": "^10.14.1",
     "firebaseui": "^6.1.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-scripts": "5.0.1",
+    "typescript": "^5.6.3",
+    "vite": "^5.4.11",
+    "vite-tsconfig-paths": "^5.1.2",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "firebase emulators:start && react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "start-without-db": "react-scripts start"
+    "serve": "vite",
+    "firebase:serve": "firebase emulators:start",
+    "start": "concurrently --names \"VITE,FIREBASE\" -c \"bgBlue.bold,bgMagenta.bold\" \"npm run serve\" \"npm run firebase:serve\"",
```

Comment:
ok this is cool
if you add --import=localdev/db i'll stamp

Reply:
deal :handshake: